Vidar/src/main/cpp/subsystems/Shooter.cpp

```diff
@@ -4,14 +4,16 @@
 
 #include "subsystems/Shooter.h"
 
-Shooter::Shooter(WPI_TalonFX& FlywheelMotor, WPI_TalonFX& HoodMotor) 
-: FlywheelMotor(FlywheelMotor)
+Shooter::Shooter(WPI_TalonFX& FlywheelMotor1, WPI_TalonFX& FlywheelMotor2, WPI_TalonFX& HoodMotor) 
```

Comment:
Curious -- Why are there Flywheel and Shooter commits in the the Feeder branch?

Putting these in a separate commit helps to justify. But we need more info why these are here and not in another branch.

See the comment in RobotContainer.h